// Copyright (c) 2023 Boston Dynamics, Inc.  All rights reserved.
//
// Downloading, reproducing, distributing or otherwise using the SDK Software
// is subject to the terms and conditions of the Boston Dynamics Software
// Development Kit License (20191101-BDSDK-SL).

syntax = "proto3";

package bosdyn.api.spot;
option go_package = "bosdyn/api/spot/inverse_kinematics_service";

option java_outer_classname = "InverseKinematicsServiceProto";

import "bosdyn/api/spot/inverse_kinematics.proto";


service InverseKinematicsService {
    rpc InverseKinematics(InverseKinematicsRequest) returns (InverseKinematicsResponse) {}
}
